test/test-index-with-dir-env.js

```diff
@@ -35,30 +36,23 @@ test.cb('CUMULUS_MESSAGE_ADAPTER_DIR sets the location of the message adapter',
   const expectedOutput = clonedeep(testContext.outputEvent);
   expectedOutput.payload = businessLogicOutput;
 
-  function callback(err, data) {
-    t.is(err, null);
-    t.deepEqual(data, expectedOutput);
-    t.end();
-  }
-
-  return cumulusMessageAdapter.runCumulusTask(businessLogic, testContext.inputEvent, {}, callback);
+  const result = await cumulusMessageAdapter.runCumulusTask(
+    businessLogic,
```

Comment:
Should businessLogic above be an async function?

Reply:
4ff5e68